app/views/cards/_header.html.erb

```diff
@@ -13,22 +13,25 @@
           <% end %>
         </div>
         <div class='ui item'>
-          <img class="ui avatar image" src="<%= avatar_url(current_user) %>">
-          <span><%= current_user.email %></span>
+          <span><%= link_to user_path(current_user), class: 'ui item' do %>
+            <img class="ui avatar image" src="<%= avatar_url(current_user) %>">
```

Comment:
Holy shit, didn't know I could use blocks to wrap links around things like this ;o